indico/web/client/js/react/components/PrincipalSearchField.jsx

```diff
@@ -139,39 +167,46 @@ export default class PrincipalSearchField extends React.Component {
 
     render() {
         const {multiple, onChange, disabled} = this.props;
-        const {isFetching, options, userId} = this.state;
+        const {isFetching, options, value} = this.state;
         let dropdownValues, selectedValues, dropdownOptions;
+        let placeholder = Translate.string('Write a name (of the group or user) or e-mail...');
```

Comment:
But that's not always the case. Unless withGroups is used the message shouldn't mention groups.